socketio/socketio.go

```diff
@@ -260,6 +266,16 @@ func InitializeSocketio(router *gin.Engine, h *server.Handlers) *socketio.Server
 						socketServer.BroadcastToRoom("/api", room, "poison_message_journey_data_"+poisonMsgId, data)
 					}
 				}
+
+				if strings.HasPrefix(room, "stations_overview_group_") && socketServer.RoomLen("/api", room) > 0 {
```

Comment:
this section should replace the factories section and to be inside the outer for loop